ext/odbc/birdstep.c

```diff
@@ -618,7 +618,7 @@ PHP_FUNCTION(birdstep_off_autocommit)
  */
 PHP_FUNCTION(birdstep_commit)
 {
-zend_long
+	zend_long id;
```

Comment:
it was changed to zend_long in 69cf6ec4 (same as all of the index variables in that file).